Makefile

```diff
@@ -370,7 +370,7 @@ local-env-setup: ## Prepare environment to run the operator with "make run"
 
 ## Miscellaneous Custom targets
 .PHONY: local-setup
-local-setup: export IMG := limitador-operator:dev
+local-setup: export IMG := localhost/limitador-operator:dev
```

Comment:
local setup was failing for me as image could not be pulled because it couldn't be found it looks like it might be a naming thing

Reply:
using podman?

Reply:
this works for docker, so we are good

Reply:
Yeah with podman